openshift/initializer-template.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: v1
+kind: Template
```

Comment:
why is this an openshift template? cant we schedule it via `argo submit ...`

Reply:
Moved to Workflow that you can submit directly